client/src/search/Typeahead/Typeahead.js

```diff
@@ -17,21 +25,26 @@ import "./Typeahead.scss";
 
 const { text_maker, TM } = create_text_maker_component(text);
 
+const cache = new CellMeasurerCache({
+  fixedWidth: true,
```

Comment:
Hmm... I wonder if that would've fixed it too now that it's being brought up again.

But I don't think it would've been worth touching anyways because warnings are thrown if there isn't a fixed width or height set (and its advised against in the documentation): 

![image](https://user-images.githubusercontent.com/25855114/110656109-a398bd00-818d-11eb-9f8a-1713a5928291.png)